components/gitpod-db/src/typeorm/project-db-impl.ts

```diff
@@ -93,4 +106,55 @@ export class ProjectDBImpl implements ProjectDB {
             await repo.save(project);
         }
     }
+
+    public async setProjectEnvironmentVariable(projectId: string, name: string, value: string): Promise<void>{
+        const envVarRepo = await this.getProjectEnvVarRepo();
+        let envVar = await envVarRepo.findOne({ projectId, name, deleted: false });
+        if (envVar) {
+            envVar.value = value;
```

Comment:
nit: this could be an `.update()` (without implicit existinance check again from typeorm). Not